src/model/openthaigpt_pretraining_model/GPTJ_TH_tokenizer/merge.py

```diff
@@ -0,0 +1,72 @@
+from transformers import AutoTokenizer, GPT2TokenizerFast
+from huggingface_hub import hf_hub_download
+from .constants import FILE_DIR, GPT2_MERGE_DIR, GPTJ_MERGE_DIR
+import json
+
+
+def merge():
+    # load tokenizer
+    gptj_tokenizer = AutoTokenizer.from_pretrained("EleutherAI/gpt-j-6B")
+    gpt2_tokenizer = AutoTokenizer.from_pretrained("flax-community/gpt2-base-thai")
+
+    # load vcab and merge rule
+    hf_hub_download(
+        repo_id="EleutherAI/gpt-j-6b",
+        filename="merges.txt",
+        local_dir=f"{FILE_DIR}/GPTJ_merge_rule/",
+    )
+    hf_hub_download(
```

Comment:
Can we make repo_id customizable `def merge(thai_tokenizer_repo: str)`

```
hf_hub_download(
        repo_id=thai_tokenizer_repo,
        filename="merges.txt",
        local_dir=f"{FILE_DIR}/GPT2_merge_rule/",
    )
```

Reply:
Done